MachObfuscator/main.swift

```diff
@@ -5,7 +5,6 @@ private func main() {
     guard !options.help,
         let appDirectory = options.appDirectory,
         let mangler = SymbolManglers.mangler(byKey: options.manglerKey) else {
-        print(Options.usage)
```

Comment:
Wyh did you remove that? I always thought printing usage description on incorrect arguments is a standard.

Reply:
Sorry - that's my mistake. Was in a hurry, thought it's my helper print.